warehouse/transformer/transformer.go

```diff
@@ -171,7 +180,6 @@ func (t *Transformer) ensureMessageID(event *types.TransformerEvent) {
 	messageID, exists := event.Message["messageId"]
 	if !exists || utils.IsBlank(messageID) {
 		event.Metadata.MessageID = "auto-" + t.uuidGenerator()
-		event.Message["messageId"] = event.Metadata.MessageID
```

Comment:
Why is `messageId` and `receivedAt` being removed?

Reply:
We don't need to modify SingularEvent. Whereever these are being used, we are using Metadata instead of Event.